config/routes.rb

```diff
@@ -1,3 +1,6 @@
 Rails.application.routes.draw do
+  get 'restaurant/index'
+
```

Comment:
Please remove the redundant (scaffolded) route

Reply:
yes, will do that